docs/content/external-data.md

```diff
@@ -136,7 +136,7 @@ That functionality is implemented using built-in functions such as [`http.send`]
 ### Current limitations
 * Unit test framework does not allow you to mock out the results of builtin functions; however, you can create a helper rule that runs the builtin and mock that out.
 * Credentials needed for the external service can either be hardcoded into policy or pulled from the environment.
```

Comment:
Would be pretty cool if it could use the http auth plugins. 💭

Reply:
Heh, I was thinking something similar the other day, but from the other side of it - I think most  of our credential plugins could now be written in Rego with the help of `http.send`, so it would be pretty cool if one could write "plug-in policies", where OPA provided whatever is needed as part of the input, and the policy would be expected to provide a certain format in return.